releasenotes/notes/apm-api-key-refresh-0aadc7d9327d8d1e.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    APM: Enable API key refreshes from secrets management on trace-agent.
```

Comment:
The feature has not be announce yet and we don't have any public  doc for this. We should omit the releasenote for now and will add one once we open the public beta.

Reply:
Oh good to know! I will remove this file then 👍